env/env_vars.go

```diff
@@ -0,0 +1,153 @@
+package env
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+type EnvironmentVar struct {
+	key      string
+	value    string
+	hasValue bool
+}
+
+var (
+	ScopeDsn                   = newEnvVar("SCOPE_DSN")
+	ScopeApiKey                = newEnvVar("SCOPE_APIKEY")
+	ScopeApiEndpoint           = newEnvVar("SCOPE_API_ENDPOINT")
+	ScopeService               = newEnvVar("SCOPE_SERVICE")
+	ScopeRepository            = newEnvVar("SCOPE_REPOSITORY")
+	ScopeCommitSha             = newEnvVar("SCOPE_COMMIT_SHA")
+	ScopeBranch                = newEnvVar("SCOPE_BRANCH")
+	ScopeSourceRoot            = newEnvVar("SCOPE_SOURCE_ROOT")
+	ScopeLoggerRoot            = newEnvVar("SCOPE_LOGGER_ROOT", "SCOPE_LOG_ROOT_PATH")
+	ScopeDisableMonkeyPatching = newEnvVar("SCOPE_DISABLE_MONKEY_PATCHING")
+	ScopeDebug                 = newEnvVar("SCOPE_DEBUG")
+	ScopeTracerGlobal          = newEnvVar("SCOPE_TRACER_GLOBAL", "SCOPE_SET_GLOBAL_TRACER")
+	ScopeTestingMode           = newEnvVar("SCOPE_TESTING_MODE")
+	ScopeTestingFailRetries    = newEnvVar("SCOPE_TESTING_FAIL_RETRIES")
+	ScopeTestingPanicAsFail    = newEnvVar("SCOPE_TESTING_PANIC_AS_FAIL")
+	ScopeConfiguration         = newEnvVar("SCOPE_CONFIGURATION")
+	ScopeMetadata              = newEnvVar("SCOPE_METADATA")
+)
```

Comment:
Yes, I think it would be better to have those specific types, right now there are 5 types: string, int, bool, slice and map

I'll make the change...

Reply:
done